code/API_definitions/location-retrieval.yaml

```diff
@@ -37,6 +37,9 @@ info:
 
     * **Max Age**: Maximum age of the location information which is accepted for the location retrieval (in seconds).
 
+      * Absence of maxAge means "any age" is acceptable for the client. In other words, this is like maxAge=infinite. In this case the system may still return lastLocationTime, if available. If the system is not able to provide location a error 400 with code LOCATION_RETRIEVAL.DEVICE_NOT_FOUND is send back.

```

Comment:
The text should refer to 404 instead of 400, I think.
I find 400 appropriate only if the maxAge is syntactically wrong.